ani.voskanyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+integer_1 = int(input("Enter the first integer:"))
+integer_2 = int(input("Enter the second integer: "))

```

Comment:
RW: User input should be in one line. Note that you should use only the techniques that you pass in the Lesson 3